src/shared/ui/paragraph/paragraph.module.scss

```diff
@@ -1,19 +1,25 @@
 .paragraph {
   margin: 0;
-  padding-bottom: 18px;
-
-  font-size: 18px;
-  font-weight: $font-weight-regular;
   line-height: 28px;
   color: $color-gray-600;
   letter-spacing: 0;
 
-  @include media-laptop {
-    font-size: 16px;
-    line-height: 24px;
+  &.medium-font-size {
+    padding-bottom: 18px;
+    font-size: 18px;
+    font-weight: $font-weight-regular;
   }
-}
 
-.top-spacing {
-  padding-top: 18px;
+  &.large-font-size {
+    padding-bottom: 16px;
+    font-size: 20px;
+    font-weight: $font-weight-medium;
+  }
+
+  @include media-laptop {
```

Comment:
I suppose, we should decrease font-size for `large-font-size` too as it was in `subtitle` earlier
![image](https://github.com/user-attachments/assets/2c87b875-fd89-4bee-a0eb-68ef589ff039)

Reply:
At the moment, our subtitles have the same font size and weight as this large variant...